.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+### Current Behavior
+
+- What code are you running and what is happening?
+- Include a screenshot if it makes sense.
+
+### Expected Behavior
+
+- What do you expect should be happening?
+- Include a screenshot if it makes sense.
+
+### How to reproduce
+
+- You must provide a way to reproduce the problem. If you are having an issue with your machine or build tools, the issue belongs on another repoistory as that is outside of the scope of this project.
```

Comment:
I'd remove this first sentence, makes it a bit more friendly.

Reply:
I would say the first sentence is the most important one. If they don't provide a way to reproduce we're just making it unnecessary difficult for the person that will try to resolve it.

Reply:
This just reads very unfriendly. I think the second sentence is enough and conveys the message.

Reply:
I think the first line reads quite (unnecessarily) unfriendly: "You must ....."

I'd either remove it or make it more friendly and welcoming.